### PR TITLE
Create new Math symbols character suite

### PR DESCRIPTION
**Pull request by srjfoo**
Create new character suite containing commonly used mathematical symbols that are not in Basic Latin.

The Math symbols character suite can be viewed here: [math-symbols](https://www.pgdp.org/~srjfoo/c/tools/charsuites.php?charsuite=math-symbols) and tested in this project: [Foundations of & wireless](https://www.pgdp.org/~srjfoo/c/project.php?id=projectID5c13a64918b30)

### CODE REVIEW

**Comment by srjfoo** [comment]
@cpeel -- I realized after I pushed the branch up that I still have the comment in place for the reference URL -- I never did get the question answered whether the wiki page [Essential Maths for Proofers](https://www.pgdp.net/wiki/DP_Official_Documentation:Proofreading/Essential_Maths_for_Proofers) should be used. I can go ahead and put it in, or just remove that comment altogether, but wanted to go ahead and get this up for review so that we can maybe get it in for Tuesday's deployment.

File: pinc/charsuite-math-symbols.inc [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+    'U+2248',
+    'U+2260-U+2261',
+    'U+2264-U+2265',
+    'U+2282-U+2283',
+    'U+2295',
+    'U+2297',
+];
+
+/*
+$charsuite->reference_urls = [
```
Comment (cpeel) [inline pinc/charsuite-math-symbols.inc:28]: I'm not sure https://www.pgdp.net/wiki/DP_Official_Documentation:Proofreading/Essential_Maths_for_Proofers is the best reference. I'd almost rather we create a similar page to https://www.pgdp.net/wiki/Symbols_collection (which could link to the Essential Maths for Proofers page) that talks about the character suite a bit more specifically.

**Review by cpeel — COMMENTED** [review]
Note to self: update https://github.com/DistributedProofreaders/dproofreaders/pull/839 after this merges in.

File: pinc/charsuite-math-symbols.inc (hunk 1 above)
Comment (srjfoo) [reply pinc/charsuite-math-symbols.inc:28]: I can do that for this PR and just put an "Under construction" message in it for a couple of days, if that works?

Comment (cpeel) [reply pinc/charsuite-math-symbols.inc:28]: That works perfectly 👍

**Review by cpeel — APPROVED** [review]

**Review by chrismiceli — APPROVED** [review]

**Pull request merged**